docs/html/reference/pip_show.rst

```diff
@@ -25,10 +25,50 @@ Options
 .. pip-command-options:: show
 
 
+Format
+======
+
+The various fields in the output and their  explanation is as follows. Note that the keys in JSON format are lowercased.
+The fields below are present in the METADATA file, which follows the specification
+laid down in `PEP-241`_, `PEP-314`_ and `PEP-345`_, with the same name as the fields, unless otherwise noted
+
+The following fields are exposed by default.
+
+- Name: The name of the distribution.
+- Version: A string containing the distribution's version number.
+- Summary: A one-line summary of what the distribution does.
+- Home-page: A string containing the URL for the distribution's home page.
+- Author: A string containing the author's name at a minimum; additional contact information may be provided.
+- Author-email: A string containing the author's e-mail address.
+- License: Text indicating the license covering the distribution where the license is not a selection from the "License" Trove classifiers.
+  This field may also be used to specify a particular version of a license which is named via the Classifier field, or to indicate a variation or exception to such a license.
+- Location: A string containing the path where the distribution is installed.
+- Requires: Each entry contains a string describing some other module or package required by this package. (Requires-Dist in METADATA file)
+- Required-by: Each entry contains a string describing some dependency in the system that the distribution is to be used.
+  This is fetched by listing all distributions for which this distribution is in Requires-Dist of their METADATA file.
+
+In verbose mode you also have access to:
```

Comment:
Regarding json output, verbose mode should not make a difference.

Reply:
Agreed. Have removed the wording.